content/docs/components/jupyter.md

```diff
@@ -4,139 +4,6 @@ description = "Using Jupyter notebooks in Kubeflow"
-
-* pods
-* deployments
-* services
-* jobs
-* tfjobs
-* pytorchjobs
-
-This means that you can directly create these k8s resources directly from your jupyter notebook. kubectl is already installed in the notebook, so you can create k8s resources running the following command in a jupyter notebook cell
-
-```
-!kubectl create -f myspec.yaml
-```
-## Creating a custom Jupyter image
-You can create your own Jupyter image and use it in your Kubeflow cluster.
-Your custom image needs to meet the requirements created by Kubeflow Notebook Controller. Kubeflow Notebook Controller  manages the life-cycle of notebooks.
- Kubeflow Web UI expects the Jupyer to be launched upon running the docker image with only `docker run`. For that you need to set the default command of your image to launch Jupyter. The Jupyter launch command needs to be set as follows:
-
-* Set the working directory: `--notebook-dir=/home/jovyan`. This is because the folder `/home/jovyan` is backed by Kubernetes Persistent Volume (PV)
-* Allow Jupyter to listen on all IPs: `--ip=0.0.0.0`
-* Allow the user to run the notebook as root: `--allow-root`
-* Set port: `--port=8888`
-* Disable authentication. Kubeflow takes care of authentication. Use the following to allow passwordless access to Jupyter: `--NotebookApp.token=''  --NotebookApp.password=''`
-* Allow any origin to access your Jupyter: `--NotebookApp.allow_origin='*'`
-* Set base_url: Kubeflow Notebook Controller manages the base URL for the notebook server using the environment variable called `NB_PREFIX`. Your should define the variable in your image and set the value of `base_url` as follows: `--NotebookApp.base_url=NB_PREFIX`
-
-As an example your Dockerfile should contain the following:
-
-
-```
-ENV NB_PREFIX /
-
-CMD ["sh","-c", "jupyter notebook --notebook-dir=/home/jovyan --ip=0.0.0.0 --no-browser --allow-root --port=8888 --NotebookApp.token='' --NotebookApp.password='' --NotebookApp.allow_origin='*' --NotebookApp.base_url=${NB_PREFIX}"]
-```
-
-## Building docker images from Jupyter Notebook on GCP
-
-If using Jupyter Notebooks on GKE, you can submit docker image builds to Cloud Build which builds your docker images and pushes them to Google Container Registry.
-
-Activate the attached service account using
-
-```
-!gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-```
-
-If you have a Dockerfile in your current directory, you can submit a build using
-
-```
-!gcloud container builds submit --tag gcr.io/myproject/myimage:tag .
-```
-
-Advanced build documentation for docker images is available [here](https://cloud.google.com/cloud-build/docs/quickstart-docker)
-
-
+Your Kubeflow deployment includes support for spawning and managing Jupyter
+notebooks. See how to [set up your notebooks](/docs/notebooks/setup/) and
```

Comment:
In this context, the Jupyter Notebook application is being spawned and managed. Should "Jupyter notebooks" be "Jupyter Notebook"? For example:

Your Kubeflow deployment includes support for spawning and managing Jupyter Notebook instances.

or

Your Kubeflow deployment includes support for spawning and managing Jupyter Notebook servers.

Reply:
I think it's OK to be less specific here, for the sake of easy reading. :)